store/models/common_test.go

```diff
@@ -144,6 +144,25 @@ func TestWebURL_MarshalJSON(t *testing.T) {
 	assert.Equal(t, `"`+str+`"`, string(b))
 }
 
+func TestWebURL_String_Has_URL(t *testing.T) {
+	t.Parallel()
+
+	u, _ := url.Parse("http://www.duckduckgo.com")
+	w := models.WebURL{
+		URL: u,
+	}
+
+	assert.Equal(t, "http://www.duckduckgo.com", w.String())
+}
+
+func TestWebURL_String_Has_Nil_URL(t *testing.T) {
```

Comment:
Let's delimit by `Test<type>_<method>_<scenario>`, so this would be `TestWebURL_String_HasNilUrl`. More on this here: https://blog.golang.org/examples

![image](https://user-images.githubusercontent.com/635121/37940466-bcc4e858-3136-11e8-8f9f-d0e28296fde2.png)

Reply:
@dimroc updated via amended commit